source/Exported_Names.html

```diff
@@ -204,20 +204,20 @@ <h2 class="subsection-title">Exported Classes</h2>
       <h2 class="subsection-title">Exported Functions</h2>
       <p>One hundred sixty-nine names exported from the Dylan module are bound to functions.</p>
       <pre class="NF.NameEntryFirst">
-\*
-\+
-\-
-\/
-\&lt;
-\&lt;=
-\=
-\==
-\&gt;
-\&gt;=
-\^
-\~
-\~=
-\~==
+*
```

Comment:
Treating these operators as names usually does require an escape backslash, so I think it would be best to keep them.

Reply:
I think I must disagree.  This section is specifically listing names, not talking about how they might be imported or exported, and it could be confusing to a newcomer as to whether the name includes the backslash or not.

(It could be confusing when they get to the point of importing these names individually, too, but chances are they will never have to do that.)

Furthermore, we don't do it for other names such as `:=` and `for`, which IIRC will have the same problem.